README.md

```diff
@@ -51,7 +51,7 @@ To use the example associated with the latest stable release, run:
 
 ### Option 2: Use the latest main branch under development
 
-Optimum Habana is a fast-moving project, and you may want to install it from source and get the latest scripts :
+Optimum Intel(R) Gaudi(R) accelerator is a fast-moving project, and you may want to install it from source and get the latest scripts :
```

Comment:
Here I would prefer to keep the same formulation as the title of the README. It is supposed to mention the name of the library and that's fine if we want to remove the "Habana" mention but I feel like "Optimum Intel(R) Gaudi(R) accelerator" doesn't make much sense as it has never been advertised that way. So I would rather have "Optimum for Intel(R) Gaudi(R) accelerator".
```suggestion
Optimum for Intel(R) Gaudi(R) accelerator is a fast-moving project, and you may want to install it from source and get the latest scripts :
```